test/Npgsql.Tests/ExceptionTests.cs

```diff
@@ -42,13 +42,15 @@ public void Basic()
             Assert.That(ex.InvariantSeverity, Is.EqualTo("ERROR"));
             Assert.That(ex.SqlState, Is.EqualTo("12345"));
             Assert.That(ex.Position, Is.EqualTo(0));
+            Assert.That(ex.Message, Is.EqualTo("12345: testexception"));
```

Comment:
I found that if I simply remove the PostgresException.Message override all existing ExceptionTests tests passed - Message was not checked.

Reply:
Thanks, good catch.